tests/page/locator-convenience.spec.ts

```diff
@@ -103,9 +103,9 @@ it('textContent should be atomic', async ({ playwright, page }) => {
       return result;
     }
   });
-  await playwright.selectors.register('textContent', createDummySelector);
+  await playwright.selectors.register('textContentFromLocators', createDummySelector);
```

Comment:
since Android shares the page, we need to use a different name. We already have this selector engine name in another test file.